src/bindings/c/include/c_api/ie_c_api.h

```diff
@@ -73,1092 +73,1128 @@
-INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode)
-ie_core_read_network(ie_core_t* core, const char* xml, const char* weights_file, ie_network_t** network);
-
-/**
- * @brief Reads the model from an xml string and a blob of the bin part of the IR. Use the ie_network_free() method to
- * free memory.
- * @ingroup Core
- * @param core A pointer to the ie_core_t instance.
- * @param xml_content Xml content of the IR.
- * @param xml_content_size Number of bytes in the xml content of the IR.
- * @param weight_blob Blob containing the bin part of the IR.
- * @param network A pointer to the newly created network.
- * @return Status code of the operation: OK(0) for success.
- */
-INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode)
-ie_core_read_network_from_memory(ie_core_t* core,
-                                 const uint8_t* xml_content,
-                                 size_t xml_content_size,
-                                 const ie_blob_t* weight_blob,
-                                 ie_network_t** network);
-
-/**
- * @brief Creates an executable network from a network previously exported to a file. Users can create as many networks
- * as they need and use them simultaneously (up to the limitation of the hardware resources). Use the
- * ie_exec_network_free() method to free memory.
- * @ingroup Core
- * @param core A pointer to the ie_core_t instance.
- * @param file_name A path to the location of the exported file.
- * @param device_name A name of the device to load the network to.
- * @param config Device configuration.
- * @param exe_network A pointer to the newly created executable network.
- * @return Status code of the operation: OK(0) for success.
- */
-INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode)
-ie_core_import_network(ie_core_t* core,
-                       const char* file_name,
-                       const char* device_name,
-                       const ie_config_t* config,
-                       ie_executable_network_t** exe_network);
-
-/**
- * @brief Creates an executable network from a network previously exported to memory. Users can create as many networks
- * as they need and use them simultaneously (up to the limitation of the hardware resources). Use the
- * ie_exec_network_free() method to free memory.
- * @ingroup Core
- * @param core A pointer to the ie_core_t instance.
- * @param content A pointer to content of the exported network.
- * @param content_size Number of bytes in the exported network.
- * @param device_name A name of the device to load the network to.
- * @param config Device configuration.
- * @param exe_network A pointer to the newly created executable network.
- * @return Status code of the operation: OK(0) for success.
- */
-INFERENCE_ENGINE_C_API(IE_NODISCARD IEStatusCode)
-ie_core_import_network_from_memory(ie_core_t* core,
-                                   const uint8_t* content,
-                                   size_t content_size,
+            typedef struct ie_network ie_network_t;
```

Comment:
@andreyanufr, can you check the code alignment in this file? It does not seem correct. Git cannot reflect the changes as well.